diagnostic.md

```diff
@@ -6,38 +6,38 @@ Write your answers inside this file, where it's indicated by the comments.
     What command do we use to do that? Use the correct URL for your fork of this
     repository in your answer.
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
+    ```
+    git clone https://github.com/tina513/git-basics-diagnostic.git
     ```
 
 2.  What do you do after cloning a repository, but before starting work?
 
-    <!-- Remove this comment and place your answer here. -->
+    create a new branch, and then switch to that branch for working.
 
 3.  What command do we use to create a new branch? Name this branch `response`
     in your answer.
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
+    ```
+   git branch response
     ```
 
 4.  We just wrote some code. What command do we use to see a summary of the
     changes in our working directory?
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
+    ```
+     git status
     ```
 
 5.  We want to prepare a change for a commit by adding a file to the staging
     area. What command do we use? Suppose the change is in the current working
     directory and named `diagnostic.md`.
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
+    ```
+    git add diagnostic.md
     ```
 
 6.  Should you ever edit published history?
 
-    ```sh
-    <!-- Remove this comment and place your answer here. -->
+    ```
+    no
```

Comment:
Ok. Thanks for your info. I will pay extra attention on those info.


Thank you~


________________________________
From: Brian Berzellini <notifications@github.com>
Sent: Wednesday, May 18, 2016 8:15 AM
To: ga-wdi-boston/git-basics-diagnostic
Cc: tina513; Author
Subject: Re: [ga-wdi-boston/git-basics-diagnostic] Git diagnostic from tina (#84)


In diagnostic.md<https://github.com/ga-wdi-boston/git-basics-diagnostic/pull/84#discussion_r63691646>:

>      ```
>
>  6.  Should you ever edit published history?
>
> -    ```sh
> -    <!-- Remove this comment and place your answer here. -->
> +    ```
> +    no


You're correct that it didn't specifically specify which format of the URL to use but that was part of the diagnostic. Given all of our clone are SSH, that's what should have been provided.

-
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/ga-wdi-boston/git-basics-diagnostic/pull/84/files/b39a80a65bb583bc13c68556d849d65f13a1b476#r63691646>